editor/svgutils.js

```diff
@@ -634,6 +634,13 @@ svgedit.utilities.cleanupElement = function(element) {
 		'ry':0
 	};
 
+	var nodeName = element.nodeName ? element.nodeName : '';
+	if (nodeName == 'ellipse') {
+		// Ellipse elements requires rx and ry attributes
```

Comment:
I'm afraid I don't understand what this PR is doing. cleanupElement removes the rx and ry attributes if they are set to zero. This PR changes cleanupElement to leave rx and ry as zero--won't the rendering still be disabled since the default rx and ry values are zero?

Can you describe how to reproduce the bug that's being fixed?